### ISSUE TITLE
Button closing PWA prompt should have aria-label

### ISSUE THREAD

**Issue by web-padawan** [vaadin/flow#7610]
The "close" button in PWA install prompt is not accessible:

https://github.com/vaadin/flow/blob/248c5c39f25f73504d3f30315f407a58d113072d/flow-server/src/main/resources/com/vaadin/flow/server/default-pwa-prompt.html#L9-L13

Can be reproduced by running Lighthouse audit for Bakery:

![a11y](https://user-images.githubusercontent.com/10589913/74663023-ad770900-51a3-11ea-9f87-4151d4ba3e0a.png)

An easy fix should be to add `aria-label="Dismiss"` (or "Close").

**Comment by knoobie** [comment]
This easy fix (adding only aria-label) is a violation of [AA Criteria of WCAG](https://www.w3.org/TR/WCAG21/#language-of-parts) if used on a page with lang != en. 

The value of the aria-label has to be configurable by the developer for i18n and accessibility.

**Comment by web-padawan** [comment]
@knoobie thanks for pointing out, that makes sense.
